src/platform-includes/configuration/config-intro/unity.mdx

```diff
@@ -16,6 +16,17 @@ You can click the button _Create options configuration_ to get the C# scriptable
 
 </Note>
 
-Note that on mobile, changes to the options object done through `ScriptableOptionsConfiguration` do not affect events coming from the native layer. Additionally, a `BeforeSend` callback will only modify events coming from C# scripts.
+The created scriptable object contains two methods that allow you to configure/override the options: 
 
-The scriptable object contains a method `Configure` that is called right before the SDK is initialized and provides you with a place to, for example, implement your own filtering of events using the [BeforeSend](/platforms/unity/configuration/filtering/#using-platformidentifier-namebefore-send-) callback.
+* `ConfigureAtBuild()` is executed by the Unity Editor during app build. 
+  Changes you make here affect current build and also allow you to alter native platform build-time settings (Android, iOS, macOS).
+
+* `ConfigureAtRuntime()` is executed on each app startup, right before the managed Sentry SDK is initialized (except for Android, iOS and macOS native layers which are already set-up by that time).
+  This can be used, for example, to implement your own filtering of events using the [BeforeSend](/platforms/unity/configuration/filtering/#using-platformidentifier-namebefore-send-) callback (note: this callback only runs for events coming from C# scripts).
+
+<Note>
+
+Not all options are used on all platforms, and some may need to be changed at build-time, other at runtime. 
+Usually, it's safe for both functions to run the same code, unless you know some options should only be set in one or the other.
+
+</Note>
```

Comment:
I've accepted the suggestion manually because there were other changes made to the docs (due to functionality changes in the respective SDK PR).

As for `Unity Editor` vs `Unity editor`, I've kept uppercases as that is the full name of the IDE